node-project-template/test/.eslintrc.json

```diff
@@ -1,8 +1,11 @@
 {
+  "plugins": ["jasmine"],
+  "extends": "plugin:jasmine/recommended",
   "env": {
     "jasmine": true
   },
   "rules": {
-    "no-console": 0
+    "no-console": 0,
+    "jasmine/no-global-setup": 0
```

Comment:
`_helpers/matchers.js` has a "global" `beforeEach` which this lint rule would complain about.